### ISSUE TITLE
Make it work on nodes too

### ISSUE THREAD

**Issue by tacotexmex** [Cahrs/w_api#3]
I imagine that the raycast functionality can be used on nodes too. Think of tools with raycast-based impact, like slicing rifts in softer nodes.

**Comment by Cahrs** [comment]
Yes its entirely possible, the problem is how should it be done? You would no longer be able to use the default node-mining system. Perhaps using metadata each swing could damage a node based on the nodes and tools properties and write the dmg change to meta (and destroying the block when reaching a threshold)?
